recipes/ncbi-amr/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.04" %}
+{% set sha256 = "da2f888c0f76fc1e99ddb6d337c4cd0e1b41c7a89397d3db1698f1d505d0deb5" %}
+
+package:
+  name: ncbi-amr
+  version: {{ version }}
+
+source:
+  url: https://github.com/ncbi/amr/archive/amrfinder_v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - compile-fix.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - coreutils
+  run:
+    - blast
```

Comment:
`blast >2.7` or so? The legacy 2.2 versions can be a bit different, IIRC.

Reply:
Good idea.